python-backend/app/api/mines/tailings/resources/tailings.py

```diff
@@ -56,7 +56,8 @@ def post(self, mine_tailings_storage_facility_guid=None):
             db.session.add(mine_tsf)
             if is_mine_first_tsf:
                 try:
-                    req_documents_url = get_documents_svc_url('/required?category=MINE_TAILINGS')
+                    req_documents_url = get_documents_svc_url('/required?category=TSF')
+                    current_app.logger.debug(req_documents_url)
```

Comment:
Is this output for monitoring? Or can this be removed?

Reply:
Removed, good catch.